From 31d3dd634d2c0560c6604c5dc72a3202e2eb282d Mon Sep 17 00:00:00 2001
From: Alexander Duyck <alexander.h.duyck@redhat.com>
Date: Wed, 22 Oct 2014 16:56:27 -0400
Subject: [redhat] configs: enable CONFIG_IP_VS_DEBUG in debug kernel

Message-id: <20141022165609.27349.49805.stgit@ahduyck-vm-rhel7>
Patchwork-id: 98068
O-Subject: [RHEL7.1 net PATCH] netfilter/ip_vs: enable CONFIG_IP_VS_DEBUG in debug kernel
Bugzilla: 1083595
RH-Acked-by: Daniel Borkmann <dborkman@redhat.com>
RH-Acked-by: Florian Westphal <fwestpha@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

This patch enables CONFIG_IP_VS_DEBUG in the debug kernels.  This is needed
in order to enable the debug_level entry in sysfs for debugging.

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1083595
Brew: http://brewweb.devel.redhat.com/brew/taskinfo?taskID=8145580

Signed-off-by: Alexander Duyck <alexander.h.duyck@redhat.com>
Signed-off-by: Jiri Benc <jbenc@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/configs/debug/CONFIG_IP_VS_DEBUG b/redhat/configs/debug/CONFIG_IP_VS_DEBUG
new file mode 100644
index 0000000..0c31fcf
--- /dev/null
+++ b/redhat/configs/debug/CONFIG_IP_VS_DEBUG
@@ -0,0 +1 @@
+CONFIG_IP_VS_DEBUG=y
-- 
1.7.1